types/sshloglevel.pp

```diff
@@ -0,0 +1,3 @@
+# The possible openssh LogLevel values. Note that these are values allowed by the
+# smart_proxy_remote_execution_ssh gem and are not consistent with the ssh_config(5) man page.
+type foreman_proxy::sshloglevel = enum['debug', 'info', 'error', 'fatal']
```

Comment:
```suggestion
type Foreman_proxy::Sshloglevel = Enum['debug', 'info', 'error', 'fatal']
```

Reply:
Yes, I was overzealous with downcasing in my text editor...